### PR TITLE
WIP need to add feedback

### PR DESCRIPTION
**Pull request by loriculberson**

### CODE REVIEW

File: students/lori_culberson/20150814-lori_culberson-m4.markdown [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## Community
+
+
+### Giving Feedback
+
+(feedback from me)
+
+### Playing a Part
+
```
Comment (kristinabrown) [inline students/lori_culberson/20150814-lori_culberson-m4.markdown]: Lori! You also started the tradition of leading a class for incomers, and that was huge, that helped me a lot coming in. You are such a strong person, being in the same cohort as you for the last couple modules has been inspirational. :tiger2:

**Comment by MichelleGolle** [comment]
Portfolio looks great! I've had a great time working with you and your energy and dedication is always motivational.

**Comment by blocknroll** [comment]
It's been really great getting to work with you and getting to know you, Lori!  You're always ready to put in the hard work and effort on our projects.  And as I've mentioned to you – I think you should teach a class on reaching out to mentors and making the most of your network.  You've been a great inspiration for lots of people at Turing and I'm so happy I've had the chance to spend time with you!

**Comment by patrickmedaugh** [comment]
Lori you work so hard and have so much heart. It's really inspiring just to see how much you care and how strong you are. It brings us all up. I loved working with you the few times we paired together.

**Comment by kylehenson** [comment]
Lori: There is nobody that can match your willingness to learn. You have the people skills, the intelligence, and the humility to approach every problem and come out on the other side with a solution. There are no limits to what you will accomplish!

**Pull request merged**